edge-modules/iotedge-diagnostics-dotnet/src/Program.cs

```diff
@@ -85,52 +73,31 @@ static async Task EdgeAgent(string managementUri)
 
         static async Task Upstream(string hostname, string port, string proxy)
         {
+            var httpClientHandler = new HttpClientHandler();
+            httpClientHandler.ServerCertificateCustomValidationCallback = (message, cert, chain, sslPolicyErrors) => {
+                    return true;   //Is valid
+            };
+
             if (proxy != null)
             {
-                Uri proxyUri = new Uri(proxy);
```

Comment:
It might be better to just have edgelet set this as an env variable in docker instead of a parameter, but if the rust changes are waiting until rc2 this can be changed then.

Reply:
+1 on that